src/api_recievers/influxqueries.js

```diff
@@ -308,4 +308,50 @@ const grabAllPreviousStudySessionIDs = () => {
 };
 
 
-module.exports = { appData, SpecificStudySessionProcessing, grabTimesForApp, insertStudySessionData, grabAllPreviousStudySessionIDs, connectToInflux };
\ No newline at end of file
+const deleteStudySession = async (sessionId) => {
+  const deleteAPI = new DeleteAPI(influxClient);
+
+  const start = '1970-01-01T00:00:00Z'; // Start of the range (earliest possible time)
+  const stop = new Date().toISOString(); // End of the range (now)
+  const deleteKey = `_measurement="AppChange" AND QuerySession == ${sessionId}`;
```

Comment:
Isn't AppChange only the name for the measurements in the WebsiteData bucket? Maybe this is why it was being really finnicky with deleting the previous sessions.

I also just remembered that I changed the name of the tag from QuerySession to sessionId for the previous study sessions bucket, which might make this query not work. We should probably just change it back to QuerySession.

Reply:
Yes I removed the measurement actually since we're doing it for all buckets now. 

Not sure what you mean by the last paragraph, if you could clarify.